bigquery_etl/cli/view.py

```diff
@@ -207,9 +208,11 @@ def publish(
         for view in views:
             view.labels["managed"] = ""
     if not force:
+        has_changes = partial(_view_has_changes, target_project)
+
         # only views with changes
-        with ThreadPool(parallelism) as p:
-            changes = p.map(lambda v: v.has_changes(target_project), views, chunksize=1)
+        with Pool(parallelism) as p:
+            changes = p.map(has_changes, views)
```

Comment:
I wonder if removing `chunksize=1` may have also contributed to the speed-up, as it will now preemptively divide up the list between the workers rather than giving each worker an item at a time.

If so, we may want to look for other cases where `chunksize=1` is being specified but the order the work is done in doesn't really matter, and try removing `chunksize=1` there as well.

Reply:
It seems to make a small difference. It was a little bit (a few seconds) faster when testing it out locally.